src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -1,146 +1,214 @@
-      <article v-if="isBenBcCccUlcCorrectionFiling" class="section-container">
-        This application must include the following:
+    <!-- Restoration conversion and extension add buttons -->
+    <div v-if="(isLimitedConversionRestorationFiling || isLimitedExtendRestorationFiling)">
+      <article>
+        <header>
+          <h2 id="resto-header-lbl">1. Add Applicant Information</h2>
+        </header>
+        <section class="mt-4">
+          <h3>Your application must include one of the following:</h3>
+        </section>
         <ul>
           <li>
-            <v-icon v-if="haveMinimumDirectors" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else color="red" class="dir-invalid">mdi-close</v-icon>
-            <span class="ml-2">
-              <template v-if="isBcCompany || isBenefitCompany || isBcUlcCompany">At least one Director</template>
-              <template v-if="isBcCcc">At least three Directors</template>
-            </span>
+            <v-icon v-if="hasApplicant" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else color="red">mdi-close</v-icon>
+            <!-- <v-icon>mdi-circle-small</v-icon> -->
+            <span>An individual</span>
+          </li>
+          <li>
+            <v-icon v-if="hasApplicant" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else color="red">mdi-close</v-icon>
+            <!-- <v-icon>mdi-circle-small</v-icon> -->
+            <span>A business or a corporation</span>
           </li>
         </ul>
-      </article>
-
-      <!-- Correction section (base corrections only) -->
-      <article v-if="isBenBcCccUlcCorrectionFiling" class="section-container">
-        <v-btn
-          id="btn-add-person"
+        <div class="mt-8">
+          <v-btn
+          id="resto-btn-add-person"
           outlined
           color="primary"
-          :disabled="isAddingEditingOrgPerson"
-          @click="initAdd([{ roleType: RoleTypes.DIRECTOR }], PartyTypes.PERSON)"
+          :disabled="this.hasApplicant"
+          @click="initAdd(
+            [{ roleType: RoleTypes.APPLICANT, appointmentDate: appointmentDate}],
+            PartyTypes.PERSON
+          )"
         >
           <v-icon>mdi-account-plus</v-icon>
           <span>Add a Person</span>
         </v-btn>
+        <v-btn
+          id="resto-btn-add-corp"
+          outlined
+          color="primary"
+          class="ml-2"
+          :disabled="this.hasApplicant"
+          @click="initAdd(
+            [{ roleType: RoleTypes.APPLICANT, appointmentDate: appointmentDate }],
+            PartyTypes.ORGANIZATION
+          )"
+        >
+          <v-icon>mdi-domain-plus</v-icon>
+          <span>Add a Business or Corporation</span>
+        </v-btn>
+        <!-- <p v-if="!hasApplicant" class="error-text small-text mt-5 mb-0">
+          You must have one applicant
+        </p> -->
```

Comment:
Ketaki, why is this commented out -- not needed or not working as desired?

Reply:
I will remove once confirm we do not need it with Ethan.

Reply:
Does this assume we have the original applicant? What if the API didn't provide it, or the user deleted it -- does this code still function correctly? (fail gracefully)

Reply:
If Removed, show red cross icon (invalid) instead of displaying above message. I just need to confirm with Ethan and remove it

![image](https://user-images.githubusercontent.com/116035339/224831073-8256a6a5-5c31-4917-b1f2-f172403b3a61.png)

Reply:
Got it. Sounds good. I'm leaving this with you.

Resolving this conversation.